todo_app/app.py

```diff
@@ -48,17 +60,20 @@ def add_item():
         todo_item_status = request.form.get('status')
```

Comment:
Your Module 10 submission was referred (and you haven't resubmitted it in Aptem yet).

At least one reason for the referral is that these routes aren't protected. Anyone can still send a POST request to your app and add a new item.

If you finish module 10, so that this route requires login and blocks non-writers, it would be a good place to add some extra logging. Log a warning (rather than info), because it's important to find out if any users are trying to perform unauthorised actions.